functional-tests/test-bed/pom.xml

```diff
@@ -23,7 +23,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <scope>test</scope>
+      <scope>compile</scope>
```

Comment:
We don't have to be explicit - `compile` is the default scope when not specified.